api/CampusService.proto

```diff
@@ -24,6 +24,117 @@ service Campus {
       response_body: "sources"
     };
   }
+
+  rpc SearchRooms (SearchRoomsRequest) returns (SearchRoomsReply) {
+    option (google.api.http) = {
+      get: "/roomfinder/room/search/{query}",
+      response_body: "rooms"
+    };
+  }
+
+  rpc GetLocations (GetLocationsRequest) returns (GetLocationsReply) {
+    option (google.api.http) = {
+      get: "/locations/{location}",
+      response_body: "locations"
+    };
+  }
+
+  rpc GetAvailableMaps (GetAvailableMapsRequest) returns (GetAvailableMapsReply) {
+    option (google.api.http) = {
+      get: "/roomfinder/room/availableMaps/{arch_id}",
+      response_body: "maps"
+    };
+  }
+
+  rpc GetCoordinates (GetCoordinatesRequest) returns (GetCoordinatesReply) {
+    option (google.api.http) = {
+      get: "/roomfinder/room/coordinates/{arch_id}",
+    };
+  }
+
+  rpc GetRoomSchedule (GetRoomScheduleRequest) returns (GetRoomScheduleReply) {
+    option (google.api.http) = {
+      get: "/roomfinder/room/scheduleById/{room_id}/{start}/{end}",
+      response_body: "events"
+    };
+  }
+}
+
+message GetRoomScheduleRequest {
+  int32 room_id = 1;
+  string start = 2; // yyyyMMdd
+  string end = 3; // yyyyMMdd
+}
+
+message GetRoomScheduleReply {
+  message RoomScheduleEvent {
+    google.protobuf.Timestamp start = 1;
+    google.protobuf.Timestamp end = 2;
+    string title = 3;
+    string event_id = 4;
```

Comment:
Is this really a string?

Reply:
int in sql, string in the current API :crossed_swords: